### DISCUSSION TITLE
[FEATURE] Add new shipping carriers Parcel Freight Logistics (PFL) and UniUni

### DISCUSSION THREAD

**Discussion by jg899** [etsy/open-api#671] [Ideas]
Hello, 

I would like to request to add PFL and UniUni to the list of carriers that can be trackable on Etsy. 

Example: 

Parcel Freight Logistics (PFL) 
https://www.aftership.com/track/pflogistics/NEXAU1007404718YQ

UniUni
https://www.uniuni.com/tracking/?no=JY22B8100002044502

Thank you

**Comment by jg899** [comment]
Hi @etsyachristensen

Is it possible to add these carriers as well to your list? [UniUni](https://www.uniuni.com/) is a major Canadian low-cost carrier that ships over 15M+ packages per year. It would provide a seamless experience for customers as currently, we have to message the tracking link to customers.

**Comment by etsyachristensen** [comment]
In order to support new shipping carriers, we have to have some specific information. Often it's easiest to have the carrier contact us to discuss adding them. But lacking that, we'd have to get the following information:

1. Example URLs for several shipments. We need to see what the url structure looks like such that we can direct users to a common url with a tracking code. For example: `https://track.somecarrier.com/?tracking=xxxxxxxxx`

2. Example of some tracking codes (they can be fake, but must match the typical formatting of a tracking code for the carrier).

We use `Aftership` as a common tool for interfacing with many carriers. If they already work with `Aftership` it should be simple enough to add them to our system. If they do not, we have to have the above information to proceed.
